blockmanager.go

```diff
@@ -710,15 +711,14 @@ func (b *blockManager) getUncheckpointedCFHeaders(
 	// Ban any peer that responds with the wrong prev filter header.
 	for peer, msg := range headers {
 		if msg.PrevFilterHeader != *filterTip {
-			log.Infof("Banning peer=%v for invalid prev filter "+
-				"header", peer)
-
+			err := b.server.BanPeer(peer, banman.InvalidFilterHeader)
```

Comment:
Should either retain all the logging in-line, or move it into the `BanPeer` method.

Reply:
Same goes for all the removed log statements in this commit.

Reply:
The log statements already exist within the `ChainService.BanPeer` wrapper method.